_posts/en/newsletters/2021-08-25-newsletter.md

```diff
@@ -0,0 +1,228 @@
+  David A. Harding notes that while not currently supported, third-party
+  purchased liquidity using [liquidity advertisements][news159 liquidity
+  advertisements] and [dual funded][topic Dual funding] channels is possible. He
+  summarizes Lisa Neigut's approaches of an onchain PSBT workflow and an
+  offchain workflow.
+
+- [Are there risks to using the same private key for both ECDSA and Schnorr signatures?]({{bse}}107924)
+  Pieter Wuille notes that while there are no known key reuse attacks across
+  [schnorr][topic schnorr signatures] and ECDSA, that "To stay within the realm of provably security it's advisable to make sure ECDSA key and Schnorr keys use distinct hardened derivation steps."
+
+## Preparing for taproot #10: PTLCs
+
+*A weekly [series][series preparing for taproot] about how developers
+and service providers can prepare for the upcoming activation of taproot
+at block height {{site.trb}}.*
+
+{% include specials/taproot/en/09-ptlcs.md %}
+
+## Releases and release candidates
+
+*New releases and release candidates for popular Bitcoin infrastructure
+projects.  Please consider upgrading to new releases or helping to test
+release candidates.*
+
+- [Rust-Lightning 0.0.100][] is a new release that supports sending and
+  receiving [keysend payments][topic spontaneous payments] and makes it
+  easier to track successfully routed payments and record the amount of
+  fee income the node earned from them.
+
+- [Bitcoin Core 22.0rc2][bitcoin core 22.0] is a release candidate
+  for the next major version of this full node implementation and its
+  associated wallet and other software. Major changes in this new
+  version include support for [I2P][topic anonymity networks] connections,
+  removal of support for [version 2 Tor][topic anonymity networks] connections,
+  and enhanced support for hardware wallets.
+
+- [Bitcoin Core 0.21.2rc1][bitcoin core 0.21.2] is a release candidate
+  for a maintenance version of Bitcoin Core.  It contains several bug
+  fixes and small improvements.
+
+## Notable code and documentation changes
+
+*Notable changes this week in [Bitcoin Core][bitcoin core repo],
+[C-Lightning][c-lightning repo], [Eclair][eclair repo], [LND][lnd repo],
+[Rust-Lightning][rust-lightning repo], [libsecp256k1][libsecp256k1
+repo], [Hardware Wallet Interface (HWI)][hwi repo],
+[Rust Bitcoin][rust bitcoin repo], [BTCPay Server][btcpay server repo],
+[Bitcoin Improvement Proposals (BIPs)][bips repo], and [Lightning
+BOLTs][bolts repo].*
+
+- [Bitcoin Core #22541][] adds a new `restorewallet` RPC command which
+  can be used to load a wallet backup. `restorewallet` complements the
+  existing `backupwallet` command which exports a copy of the currently
+  loaded wallet. Note that `restorewallet` is not compatible with
+  `dumpwallet` which exports to a human readable format that is loaded
+  using `importwallet`.
```

Comment:
When I asked for Andrew Chow's feedback, he noted that mentioning `dumpwallet` and `importwallet` was unnecessary. If you feel the same way, please feel free to remove the last sentence:

```suggestion
  loaded wallet.
```

Reply:
I think it's useful information.  I'm going to rewrite it a bit as I don't entirely like calling it "not compatible".